src/pkg/logger/logger.go

```diff
@@ -3,20 +3,26 @@ package logger
 import (
 	"context"
 	"os"
+	"time"
 
 	"github.com/spf13/cobra"
 	"github.com/spf13/pflag"
 	"go.uber.org/zap"
 	"go.uber.org/zap/zapcore"
 )
 
+var defaultLogLocation = "corso-" + time.Now().UTC().Format("2006-01-02T15-04-05Z") + ".log"
```

Comment:
Default log location is currently `$PWD/corso-<timestamp>.log`.

Reply:
Works for me.  @gmatev you happy with this?  Alternatively, we could co-locate it with the config file location by default.  Or perhaps a ~/.corso folder.

Reply:
FYI, if we are changing to a different location, we will also have to consider what it will be in Windows.

Reply:
Another option would be stick to [XDG base directory specification](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html) in linux and store it under `$XDG_STATE_HOME` which would be `$HOME/.loca/state/corso/corso-<timestamp>.log`.

As a follow-up, we should probably consider moving the config file to `$XDG_CONFIG_HOME`, which would be `$HOME/.config/corso/config.toml`.

There is a relatively  vocal group of people(myself kinda included) advocating against polluting home directory on linux: https://www.reddit.com/r/linux/comments/971m0z/im_tired_of_folders_littering_my_home_directory/

Reply:
Agree that this should not be home directory or local directory. Suggest we emulate https://kopia.io/docs/advanced/logging/